crates/ev-cli/src/theme.rs

```diff
@@ -0,0 +1,373 @@
+            write!(
+                f,
+                "{} {} ",
+                &self.success_prefix,
+                self.prompt_style.apply_to(prompt)
+            )?;
+        }
+
+        write!(
+            f,
+            "{} {}",
+            &self.success_suffix,
+            self.values_style.apply_to(sel)
+        )
+    }
+
+    /// Formats a password prompt after selection.
+    fn format_password_prompt_selection(
+        &self,
+        f: &mut dyn fmt::Write,
+        prompt: &str,
+    ) -> fmt::Result {
+        self.format_input_prompt_selection(f, prompt, "********")
+    }
+
+    /// Formats a multi select prompt after selection.
+    fn format_multi_select_prompt_selection(
+        &self,
+        f: &mut dyn fmt::Write,
+        prompt: &str,
+        selections: &[&str],
+    ) -> fmt::Result {
+        if !prompt.is_empty() {
+            write!(
+                f,
+                "{} {} ",
+                &self.success_prefix,
+                self.prompt_style.apply_to(prompt)
+            )?;
+        }
+
+        write!(f, "{} ", &self.success_suffix)?;
+
+        if self.inline_selections {
+            for (idx, sel) in selections.iter().enumerate() {
+                write!(
+                    f,
+                    "{}{}",
+                    if idx == 0 { "" } else { ", " },
+                    self.values_style.apply_to(sel)
+                )?;
+            }
+        }
+
+        Ok(())
+    }
+
+    /// Formats a select prompt item.
+    fn format_select_prompt_item(
+        &self,
+        f: &mut dyn fmt::Write,
+        text: &str,
+        active: bool,
+    ) -> fmt::Result {
+        let details = match active {
+            true => (
+                &self.active_item_prefix,
+                self.active_item_style.apply_to(text),
+            ),
+            false => (
+                &self.inactive_item_prefix,
+                self.inactive_item_style.apply_to(text),
+            ),
+        };
+
+        write!(f, "{} {}", details.0, details.1)
+    }
+
+    /// Formats a multi select prompt item.
+    fn format_multi_select_prompt_item(
+        &self,
+        f: &mut dyn fmt::Write,
+        text: &str,
+        checked: bool,
+        active: bool,
+    ) -> fmt::Result {
+        let details = match (checked, active) {
+            (true, true) => (
+                &self.checked_item_prefix,
+                self.active_item_style.apply_to(text),
+            ),
+            (true, false) => (
+                &self.checked_item_prefix,
+                self.inactive_item_style.apply_to(text),
+            ),
+            (false, true) => (
+                &self.unchecked_item_prefix,
+                self.active_item_style.apply_to(text),
+            ),
+            (false, false) => (
+                &self.unchecked_item_prefix,
+                self.inactive_item_style.apply_to(text),
+            ),
+        };
+
+        write!(f, "{} {}", details.0, details.1)
```

Comment:
This seems needlessly verbose
```suggestion
        let prefix = if checked { 
          self.checked_item_prefix 
        } else { 
          self.unchecked_item_prefix 
        };
        
        let prompt_item = if active { 
          self.active_item_style.apply_to(text) 
        } else { 
          self.inactive_item_style.apply_to(text) 
        };
       
        write!(f, "{} {}", prefix, prompt_item)
```